### PR TITLE
Update permissive mode description

### PR DESCRIPTION
**Pull request by wattli**
To make it more accurate

### CODE REVIEW

**Comment by wattli** [comment]
@wenchenglu FYI

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ In this option, all services, as servers, can accept both plain text and
 mutual TLS traffic. However, all services, as clients, will send plain
 text traffic.
 Visit [mutual migration](/docs/tasks/security/mtls-migration/#configure-clients-to-send-mutual-tls-traffic)
-for how to configure clients behavior.
+for how to configure clients behavior. To enforce mutual TLS authenticaiont, choose Option 2.
```
Comment (howardjohn) [inline content/docs/setup/kubernetes/quick-start/index.md]: typo - authentication

Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start/index.md]: Suggest making it a little more clear:

To enforce mutual TLS authentication for both clients and servers, choose [Option 2](#option-2-install-istio-with-default-mutual-tls-authentication) instead.

Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start/index.md]: client behavior

Comment (wattli) [reply content/docs/setup/kubernetes/quick-start/index.md]: Done.

Comment (wattli) [reply content/docs/setup/kubernetes/quick-start/index.md]: done

Comment (wattli) [reply content/docs/setup/kubernetes/quick-start/index.md]: done

**Pull request merged**